tests/ten_minutes_wakt_test.js

```diff
@@ -0,0 +1,13 @@
+import test from 'ava';
+import isValidWalk from '../src/ten_minutes_walk.js';
+
+test('Walk should be valid', t => {
```

Comment:
What is valid walk?
`
test('Person ends at starting point after 10 minutes')
`